robotino_bringup/robots/uibk-robotino2/urdf/robotino.urdf.xacro

```diff
@@ -20,11 +30,11 @@
     </plugin>
   </gazebo>
 
-  <hokuyo name="hokuyo" parent="base_link">
+  <hokuyo name="hokuyo" parent="base_linkz">
     <origin xyz="${hokuyo_x} ${hokuyo_y} ${hokuyo_z}" rpy="${hokuyo_roll} ${hokuyo_pitch} ${hokuyo_yaw}"/>
   </hokuyo>
 
-  <xacro:arm name="arm_" parent="base_link">
+  <xacro:arm name="arm_" parent="base_linkz">
```

Comment:
should be defined in the squirrel_common repo (https://github.com/shangl/squirrel_common/commits/indigo_dev) . i checked it, it indeed is not there. obviously i forgot to push squirrel_common. will check that tomorrow